src/engines/synths/potion/potion.cpp

```diff
@@ -89,23 +93,32 @@ namespace otto::engines {
   void PotionSynth::load_wavetable(int wt_number, std::string filename)
   {
     AudioFile<float> aux;
-    aux.load(Application::current().data_dir / "wavetables" / filename);
-    props.wavetables[wt_number].resize(aux.getNumSamplesPerChannel());
-    props.samplerates[wt_number] = aux.getSampleRate();
-    util::copy(aux.samples[0], props.wavetables[wt_number].elems());
+    bool loaded = aux.load(Application::current().data_dir / "wavetables" / filename);
+    int num_samples;
```

Comment:
try to avoid unintialized integral types (ints, floats etc)

Reply:
-1 line